src/Athena/Content/js/schedule.js

```diff
@@ -204,6 +204,8 @@ export function init(studentId, readOnly) {
                 .attr('data-position', 'bottom')
                 .attr('data-delay', 25)
                 .attr('data-tooltip', event.title)
+            if (!readOnly) {
+                $(element).addClass('tooltipped')
```

Comment:
This is redundant, tooltips are added earlier to the element regardless of `readonly` mode.